client/my-sites/checkout/composite-checkout/payment-method-helpers.js

```diff
@@ -348,3 +351,9 @@ export function filterAppropriatePaymentMethods( {
 			);
 		} );
 }
+
+export function getPostalCode() {
+	const countryCode = select( 'wpcom' )?.getContactInfo?.()?.countryCode?.value ?? '';
+	const postalCode = select( 'wpcom' )?.getContactInfo?.()?.postalCode?.value;
```

Comment:
Does this mean that `postalCode` can be undefined (and `tryToGuessPostalCodeFormat()` expects a string)?